README.md

```diff
@@ -75,6 +75,26 @@ var results = conn.Execute(sql.CommandText, sql.Parameters);
 var results = syntax.QueryCollection<EmployeeObj>(list);
 ```
 
+# Constructing a Query
+
+Searchlight supports most operators common to SQL, such as:
+* Equals (=, EQ)
+* Greater Than (>, GT)
+* Greater Than Or Equal (>=, GE)
+* Less Than (<, LT)
+* Less Than Or Equal (<=, LE)
+* Not Equal (!=, NE, <>)
+* In
+* Contains
+* StartsWith
+* EndsWith
+* IsNull
+* AND
+* OR
+* NONE  
+
+As well as sort directions specified by `ASC` and `DESC`, and encapsulated quotes denoted by `''` or `""` for filters like `'Category eq Metallica''s Covers'`.
```

Comment:
Typo:

`'Category eq Metallica''s Covers'`

should be

`Category eq 'Metallica''s Covers'`.